packages/core/src/redux/tests/store/localMedia.spec.ts

```diff
@@ -272,6 +275,39 @@ describe("actions", () => {
         });
     });
 
+    describe.only("doToggleLowDataMode", () => {
+        describe("when low data mode is enabled", () => {
+            let initialState: Partial<RootState>;
+
+            beforeEach(() => {
+                initialState = {
+                    localMedia: {
+                        busyDeviceIds: [],
+                        cameraEnabled: true,
+                        devices: [],
+                        isSettingCameraDevice: false,
+                        isSettingMicrophoneDevice: false,
+                        isTogglingCamera: false,
+                        lowDataMode: false,
+                        microphoneEnabled: true,
+                        status: "started",
+                        stream: new MockMediaStream(),
+                        isSwitchingStream: false,
+                    },
+                };
+            });
+
+            it("should call doSwitchLocalStream", async () => {
+                jest.spyOn(localMediaSlice, "doSwitchLocalStream");
+                const store = createStore({ initialState });
+
+                await store.dispatch(localMediaSlice.doToggleLowDataMode());
+
+                expect(localMediaSlice.doSwitchLocalStream).toHaveBeenCalledTimes(1);
```

Comment:
```suggestion
                const before = store.getState().localMedia;
                
                store.dispatch(localMediaSlice.doToggleLowDataMode());
                
                const after = store.getState().localMedia;

                expect(localMediaSlice.doSwitchLocalStream).toHaveBeenCalledTimes(1);
                expect(diff(before, after)).toEqual({
                    lowDataMode: true,
                });
```
No need for await here, the action creator is not async.
And let's assert on the state update here as well.